src/main/java/com/courtcanva/ccfranchise/services/StaffService.java

```diff
@@ -22,4 +23,12 @@ public StaffGetDto createStaff(Staff staff) {
 
     }
 
+    public Boolean emailExists(String email) {
+        Boolean isExisted = staffRepository.existsStaffByEmail(email);
+        if (isExisted) {
+            throw new ResourceAlreadyExistException("Email already existed");
```

Comment:
not here but in controlleradvisor, suggest to use HttpStatus.CONFLICT for ResourceAlreadyExistException